womtool/src/test/scala/womtool/WomtoolValidateSpec.scala

```diff
@@ -13,7 +13,6 @@ class WomtoolValidateSpec extends FlatSpec with Matchers {
 
   behavior of "womtool validate"
 
-
```

Comment:
Unless you have strong opinions on this line, I'd revert this to keep the file from showing up in the change set.

Reply:
Because IMO two-line-breaks are worthy of shrinking (but mainly to avoid having to re-run this PR through Travis) I'll leave this in unless you strongly object?